drivers/modem/gsm_ppp.c

```diff
@@ -302,7 +304,18 @@ static void set_ppp_carrier_on(struct gsm_modem *gsm)
 	}
 
 	api = (const struct ppp_api *)ppp_dev->api;
-	api->start(ppp_dev);
+	if (is_booting_phase) {
+		api->start(ppp_dev);
+	} else {
+		int status;
+		struct net_if *network_int;
```

Comment:
Please name the variable `iface` as that is how it is normally called.

Reply:
thanks for the review, addressed now